charts/alpha/keycloak/values.yaml

```diff
@@ -0,0 +1,45 @@
+keycloak:
+  networkPolicy:
+    enabled: false
+
+  postgresql:
+    enabled: false
+    primary:
```

Comment:
since we have enabled: false here already, do we actually need the `primary` overrides? My read is that the sub chart will be entirely disabled because of `enabled: false`